client/pom.xml

```diff
@@ -29,11 +29,17 @@
     </parent>
     <repositories>
         <repository>
-            <id>juniper-contrail</id>
-            <url>https://juniper.github.io/contrail-maven/snapshots</url>
+            <id>juniper-tungsten-api</id>
+            <url>https://github.com/radu-todirica/tungsten-api/raw/master</url>
```

Comment:
is this going to be moved elsewhere?

Reply:
Yes. This jar needs to be moved to a Cloudstack repository.

Reply:
New source is https://github.com/kiwiflyer/tf-java-api